docker/unified-builder

```diff
@@ -122,6 +122,14 @@ RUN apt-get update && \
   && \
   apt-get clean && rm -rf /var/lib/apt/lists/*
 
+# Download OPA binary
+ARG TARGETARCH
+ARG OPA_VERSION=0.49.0
```

Comment:
```suggestion
ARG OPA_VERSION=0.49.2
```
unless we're being careful to match other stuff that it's not worth also updating right now.

Reply:
Good point!